packages/fxa-content-server/app/tests/spec/lib/fxa-client.js

```diff
@@ -1884,4 +1884,23 @@ describe('lib/fxa-client', function () {
       });
     });
   });
+
+  describe('updateEcosystemAnonId', () => {
+    it('updates the ecosystem id', () => {
+      const sessionToken = 'the silence';
+      const ecosystemAnonId = 'the gold';
+      sinon.stub(realClient, 'updateEcosystemAnonId').resolves();
+
+      return client
+        .updateEcosystemAnonId(sessionToken, ecosystemAnonId)
+        .then((res) => {
+          assert.isTrue(
+            realClient.updateEcosystemAnonId.calledOnceWith(
```

Comment:
If you stub out the `realClient.sessionPut` would you be able to assert the correct path, method and headers?

Reply:
yeah, I'll have that updated shortly